src/components/pages/CaseTable/CaseTable.css

```diff
@@ -212,3 +212,17 @@ button.ant-btn.columnsBtn:hover {
   padding: 0;
   margin: 0;
 }
+
+/* Chart Container Adjustments */
+
+.chartContainer {
+  display: flex;
+}
+
+.divider {
+  width: 2px;
+  height: 200px;
+  background-color: grey;
+  border: 1px solid black;
+  margin-top: 2rem;
+}
```

Comment:
I would like to convert this to a less file in the future to keep the styling consistent and then you can nest the styling as well.

Reply:
You got it - I'll switch it over now and make another push!